build-packages/scandipwa-development-toolkit-core/package.json

```diff
@@ -0,0 +1,42 @@
+{
+	"version": "1.0.0",
+	"name": "@scandipwa/scandipwa-development-toolkit-core",
+	"publisher": "ScandiPWA",
+	"description": "ScandiPWA development tools' base",
+	"license": "SEE LICENSE IN LICENSE",
+	"main": "./out",
+	"scripts": {
+		"compile": "tsc -p ./ -d && cp -r src/templates out/",
```

Comment:
How to publish this one? You added some routine to compile it?

Reply:
When I pull changes, will it work withou compilation? I do not see compiled files in git